src/vos/vos_policy.c

```diff
@@ -0,0 +1,61 @@
+/**
+ * (C) Copyright 2016-2021 Intel Corporation.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause-Patent
+ */
+/**
+ * Object placement policy
+ *  Policy definitions
+ *
+ */
+
+#include <daos_srv/policy.h>
+#include "vos_policy.h"
+
+/* policy functions definitions */
+
+/* policy based on io size
+ *
+ * Storage tier is determined using size of the IO operation and user specified
+ * per pool thresholds.
+ *
+ * This policy uses configurable parameters as size thresholds.
+ * If parameters are 0, default thresholds are used.
+ */
+static enum daos_media_type_t
+policy_io_size(struct vos_pool *pool, daos_iod_type_t type, daos_size_t size)
+{
+	uint32_t		scm_threshold;
+
+	if (pool->vp_vea_info == NULL)
+		return DAOS_MEDIA_SCM;
+
+	scm_threshold = pool->vp_policy_desc.params[0] > 0 ?
+		pool->vp_policy_desc.params[0] : VOS_POLICY_SCM_THRESHOLD;
+
+	return (size > scm_threshold) ? DAOS_MEDIA_NVME : DAOS_MEDIA_SCM;
```

Comment:
I think this is wrong, it should be "size >= scm_threshold", otherwise, 4k size will be landed to SCM?